modules/objdetect/include/opencv2/objdetect/barcode.hpp

```diff
@@ -57,6 +57,52 @@ class CV_EXPORTS_W_SIMPLE BarcodeDetector : public cv::GraphicalCodeDetector
                                       CV_OUT std::vector<std::string> &decoded_info,
                                       CV_OUT std::vector<std::string> &decoded_type,
                                       OutputArray points = noArray()) const;
+
+    /** @brief Get detector downsampling threshold.
+     *
+     * @return detector downsampling threshold
+     */
+    CV_WRAP double getDownSampleThresh() const;
```

Comment:
I suggest renaming to `getDownsamplingThresh` (similarly for set-method). "downsampling" is a single world.

Reply:
Renamed.